CHANGES.md

```diff
@@ -3067,12 +3073,12 @@ OpenSSL 1.1.0
 
  * State machine rewrite. The state machine code has been significantly
    refactored in order to remove much duplication of code and solve issues
-   with the old code (see ssl/statem/README for further details). This change
-   does have some associated API changes. Notably the SSL_state() function
-   has been removed and replaced by SSL_get_state which now returns an
-   "OSSL_HANDSHAKE_STATE" instead of an int. SSL_set_state() has been removed
-   altogether. The previous handshake states defined in ssl.h and ssl3.h have
-   also been removed.
+   with the old code (see [ssl/statem/README.md](ssl/statem/README.md) for
```

Comment:
You're thinking of [link references](https://spec.commonmark.org/0.29/#link-reference-definitions).  Don't go there.

Reply:
> Isn't there a shortcut version of this, like `[foobar.md]` ? Here and below.

This would be rather ambiguous and error-prone,
and I did not find this in https://daringfireball.net/projects/markdown/syntax#link.